src/components/interaction-designer/block-editors/TagSelector.vue

```diff
@@ -17,7 +17,17 @@
         :close-on-select="false"
         :taggable="taggable"
         :tag-placeholder="taggable ? trans('flow-builder.create-a-tag-prompt') : ''"
-        @tag="addTag" />
+        @tag="addTag">
+        <template #clear>
+          <font-awesome-icon
+            v-if="selectedTags.length"
+            v-b-tooltip.hover="trans('flow-builder.clear-all')"
+            style="position: absolute; right: 41px; top: 13px; cursor: pointer"
```

Comment:
Extracting this to a class didn't work. Probably, a class becomes unavailable when passed via the slot.

Reply:
also, the 'clear' slot doesn't work well. Vue-multiselect uses 'position: absolute' here and there, its layout is not great, so we need to use 'position: absolute' too.

Reply:
Without it the `x` goes on top
<img width="480" alt="Screen Shot 2022-11-07 at 11 44 59 PM" src="https://user-images.githubusercontent.com/18130716/200442104-74569cfe-6382-4b39-b4dc-0a75f3138902.png">

Reply:
Never mind. I've used the `>>>` deep combinator and it worked